Gemfile

```diff
@@ -33,6 +33,7 @@ group :development do
   gem 'capistrano-passenger', require: false
   gem 'capistrano-pending', require: false
   gem 'capistrano-rails', require: false
+  gem 'listen'
```

Comment:
Why this?

Reply:
https://github.com/umts/ft-forms/pull/89/files?utf8=%E2%9C%93&diff=split#diff-a2f09e1db7a47eb67c5849478a97f3d7R60